CHANGELOG.md

```diff
@@ -11,6 +11,7 @@ All notable changes to this project will be documented in this file.
 -   Add ability to view the Git commit log
 -   Allow generating ECDSA and ED25519 keys for SSH
 -   Add support for multiple/fallback authentication methods for SSH
+-   Add [Kiwi](https://kiwibrowser.com/) to supported browsers list for Autofill
```

Comment:
This line shouldn't be added separately and instead Kiwi should be mentioned in the 2nd line of this section which already covers two other browsers that are now supported.